### PR TITLE
RTLCSS all the things

### PR DESCRIPTION
**Pull request by swissspidy**
See #2976.

To do:

* [x] Make sure RTL stylesheets are actually loaded by WordPress

### CODE REVIEW

File: .rtlcssrc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
```
Comment (swissspidy) [inline .rtlcssrc]: This is the default config as mentioned at https://rtlcss.com/learn/usage-guide/cli/

Without a config file there are warnings.

**Comment by westonruter** [comment]
Merge conflict in `.stylelintignore`.

**Comment by swissspidy** [comment]
Tested the changes using both the RTL Tester plugin and by temporarily switching the site language to Arabic. Visited all the admin pages, the customizer, and the frontend and everything looked good.

The only issue is with the AMP Stories editor. I've opened #3012 to address that separately.

**Comment by westonruter** [comment]
@swissspidy I'm not seeing the “Enable AMP” checkbox for some reason:

image

**Comment by swissspidy** [comment]
@westonruter I see it just fine in both Transitional and Standard mode:

Screenshot 2019-08-12 at 19 36 02

**Comment by westonruter** [comment]
I think `npm run build` may not be including some required files. I'm not seeing it on a fresh Arabic site I just spun up on Pantheon either: 

image

**Comment by westonruter** [comment]
Also noticing an AMP validation error now due to `amp-default.css` not being found:

image

**Comment by swissspidy** [comment]
_checking..._

**Comment by westonruter** [comment]
Turns out to be because Gutenberg was not active on the environment. So not related to RTL.

**Pull request merged**